vm/api/src/responses/LoadStateJsonRpcResponse.ts

```diff
@@ -0,0 +1,10 @@
+import type { LoadStateError } from '../errors/LoadStateError.js'
+import type { LoadStateResult } from '../result/LoadStateResult.js'
+import type { SerializeToJson } from '../utils/SerializeToJson.js'
+import type { JsonRpcResponse } from './JsonRpcResponse.js'
+
+export type LoadStateJsonRpcResponse = JsonRpcResponse<
+	'tevm_load_state',
```

Comment:
```suggestion
	'tevm_loadState',
```
This casing is how eth methods do it so best to stay consistent